.github/stale.yml

```diff
@@ -0,0 +1,62 @@
+# Configuration for probot-stale - https://github.com/probot/stale
+
+# Number of days of inactivity before an Issue or Pull Request becomes stale
+daysUntilStale: 60
+
+# Number of days of inactivity before an Issue or Pull Request with the stale label is closed.
+# Set to false to disable. If disabled, issues still need to be closed manually, but will remain marked as stale.
+daysUntilClose: 7
+
+# Only issues or pull requests with all of these labels are check if stale. Defaults to `[]` (disabled)
+onlyLabels: []
+
+# Issues or Pull Requests with these labels will never be considered stale. Set to `[]` to disable
+exemptLabels:
+  - "contrib: maybe good first bug"
+  - "contrib: good first bug"
+  - "contrib: welcome"
+  - "component: security"
+
+# Set to true to ignore issues in a project (defaults to false)
+exemptProjects: true
+
+# Set to true to ignore issues in a milestone (defaults to false)
+exemptMilestones: true
+
+# Set to true to ignore issues with an assignee (defaults to false)
+exemptAssignees: false
+
+# Label to use when marking as stale
+staleLabel: "state: stale"
+
+# Comment to post when marking as stale. Set to `false` to disable
+markComment: >
+  This issue has been automatically marked as stale because it has not had
+  recent activity. It will be closed if no further activity occurs. Thank you
+  for your contributions.
```

Comment:
I wonder if some kind of call to action might be worth adding, something like "If you think this issue should not be closed, please do let us know and we can re-triage."

:thinking:

Reply:
I think that's already implied no?

Reply:
Being explicit is always better than implicitly implying something. Especially for people not too familiar with the project or open source contributions often need a bit more hand-holding. I also think it's a good habit to encourage people to help us prioritize their issues.

Reply:
:+1: on @EnTeQuAk 's suggestion, I think it's not explicit enough. Maybe word that in such a way that encourages people to help in some way? A less aggressive version of "send us your patch if you think it should stay opened" :-)

Reply:
Fair enough, I think it would be better to re-word the comment, rather than try to add something to reinforce what's already there. To me  "It will be closed if no further activity occurs" is quite clear and unambiguous about what will happen. 

If we include something with a more concrete action re: having users ask for issues to be re-triaged, then we'll need to be sure that this is a process that will happen and doesn't get missed. Any thoughts on how to do that?

Reply:
Reworded to:

```yaml
markComment: >
  This issue has been automatically marked as stale because it has not had
  recent activity. If you think this bug should stay open, please comment on 
  the issue with further details. Thank you for your contributions.
```